src/core/render/index.js

```diff
@@ -330,7 +330,10 @@ export function renderMixin(proto) {
           },
           tokens => {
             html = this.compiler.compile(tokens);
-            html = this.isRemoteUrl ? DOMPurify.sanitize(html) : html;
+            // add "target" attribute to DOMPurify white list to handle external links
+            html = this.isRemoteUrl
+              ? DOMPurify.sanitize(html, { ADD_ATTR: ['target'] })
```

Comment:
so this change will skip sanitizing the attributes ?

Reply:
No, this change will consider the `target` attribute as a valid one and will not remove it. But it does not prevent DOMPurify to sanitize the attribute content. For example, if the following code is present in the source
```html
<a target="javascript:alert('XSS')" href="https://example.com">Test</a>
```
it will be changed to
```html
<a href="https://example.com">Test</a>
```
because the value will be considered as unsafe.

Reply:
cool, Can you add test(s) for this, Otherwise looks good to me

Reply:
@Fab1en So if it sees `target="_blank"` then it will leave it in that case? Mind adding a small test case?